etl/steps/data/garden/artificial_intelligence/2024-06-03/epoch_aggregates_affiliation.py

```diff
@@ -0,0 +1,75 @@
+"""Load a meadow dataset and create a garden dataset."""
+import pandas as pd
+import shared as sh
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
```

Comment:
you don't need to import `get_logger` and use `log`. Instead, simply use object `paths.log`.

So you can do `paths.log.info` instead of `log.info`.